client/my-sites/importer/error-pane.jsx

```diff
@@ -31,7 +31,7 @@ class ImporterError extends React.PureComponent {
 	contactSupport = event => {
 		event.preventDefault();
 		event.stopPropagation();
-		Page( '/help' );
+		Page( '/help/contact' );
```

Comment:
I'm in two minds about the usefulness of jumping straight to a contact form vs going to the support page about this, e.g. https://support.wordpress.com/import/#importing-content-from-another-wordpress-blog. (This redirects me to the page for my language). Would you say a contact form is more helpful?

That page didn't mention zips! I've updated it - good thing you had another look at this! 👍

Reply:
I agree -- I feel like we should send customers to the documentation first, unless there is no documentation that could be useful. Sending customers directly to support seems prone to increase the load on support.

@andfinally Thanks for updating the support doc to mention zips!

Reply:
I don't think the increased load on support would be significant given the traffic to importers, but I'm fine keeping the link as is. I will still change the original language then to refer to Help section, not support.